src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -68,6 +69,7 @@ class PhpDumper extends Dumper
     private $inlineRequires;
     private $inlinedRequires = array();
     private $circularReferences = array();
+    private $singlePrivateIds = array();
```

Comment:
Minor comment. I didn't like the `$singlePrivateIds" var name, so I looked for [English words that mean "used once"](https://english.stackexchange.com/questions/145236/a-word-describes-things-that-can-be-used-only-once) and maybe it could be better to rename this to `$singleUsePrivateIds`